.gitignore

```diff
@@ -1,10 +1,9 @@
 .vscode/
 node_modules/
 # form upload paths
-loopback/server/uploads/*
```

Comment:
we still need this

Reply:
whoops rushed that.  would be nice if we were using consistent paths for these....